src/Components/News/NewsList.js

```diff
@@ -1,32 +1,142 @@
-        {id: 2, name: 'Titulo de prueba', description: 'Descripcion de prueba'},
-        {id: 1, name: 'Titulo de prueba', description: 'Descripcion de prueba'},
-        {id: 3, name: 'Titulo de prueba', description: 'Descripcion de prueba'}
-    ];
-
-    return (
-        <div>
-            <h1>Listado de Novedades</h1>
-            <ul className="list-container">
-                {newsMock.length > 0 ? 
-                    newsMock.map((element) => {
-                        return(
-                            <li className="card-info" key={element.id}>
-                                <h3>{element.name}</h3>
-                                <p>{element.description}</p>
-                            </li>
-                        )
-                    })
-                :
-                    <p>No hay novedades</p>
-                }
-            </ul>
-        </div>
-    );
-}
- 
-export default NewsList;
\ No newline at end of file
+  const [modalImg, setModalImg] = useState({});
+
+  const newsMock = [
+    {
+      id: 1,
+      name: 'Titulo de prueba 1',
+      image:
+        'http://www.w-h-p-h.com/wp-content/uploads/2016/06/Test-Cover-01.jpg',
+      createdAt: '17/11/21',
+    },
+    {
+      id: 2,
+      name: 'Titulo de prueba 2',
+      image:
+        'http://www.w-h-p-h.com/wp-content/uploads/2016/06/Test-Cover-02.jpg',
+      createdAt: '18/11/21',
+    },
+    {
+      id: 3,
+      name: 'Titulo de prueba 3',
+      image:
+        'http://www.w-h-p-h.com/wp-content/uploads/2016/06/Test-Cover-03.jpg',
+      createdAt: '19/12/21',
+    },
+    {
+      id: 4,
+      name: 'Titulo de prueba 4',
+      image:
+        'http://www.w-h-p-h.com/wp-content/uploads/2016/06/Test-Cover-04.jpg',
+      createdAt: '20/12/21',
+    },
+  ];
+
+  const openModalImg = (src, name) => {
+    setModalImg({
+      open: true,
+      src: src,
+    });
+  };
+
+  const closeModalImg = () => {
+    setModalImg({
+      open: false,
+    });
+  };
+
+  const toEditForm = () => {
+    alert('direcciona a formulario de edicion');
+  };
+
+  const RemoveNew = () => {
+    alert('elimina la novedad');
+  };
+
+  return (
+    <Container fluid>
+      <Row>
+        <h1 className='text-center mt-4 mb-4'>Listado de Novedades</h1>
+        <Col>
+          <Button
+            className='mt-4 mb-4'
+            as={Link}
+            to={`/backoffice/news/create`}
+          >
+            Crear Novedad
+          </Button>
+        </Col>
+      </Row>
+      <Row>
+        <Col>
+          <Table hover striped bordered className='align-middle'>
+            <thead>
+              <tr>
+                <th>Name</th>
+                <th>Image</th>
+                <th>createdAt</th>
+                <th className='text-center'>Actions</th>
+              </tr>
+            </thead>
+            <tbody>
+              {newsMock.length > 0 ? (
+                newsMock.map((element) => {
+                  return (
+                    <tr>
```

Comment:
Guarda, falta el atributo key de los ítems de la lista.
https://reactjs.org/docs/lists-and-keys.html

Reply:
Atributo key agregado.